js/tools/randAlpha.js

```diff
@@ -0,0 +1,44 @@
+/* global app */
+window.tools.randAlpha = {
+  name: 'randAlpha',
+  icon: '/images/randAlpha.png',
+  state: {
+    selected: false,
+    mousePresesd: false
+    // other state properties go here
+  },
+  events: {
+    mousedown: function (e, self) {
+      self.state.mousePressed = true
+    },
+    mouseup: function (e, self) {
+      self.state.mousePressed = false
+    },
+    mousemove: function (e, self) {
+      // if self tool is selected AND the mouse is pressed
+      if (self.state.selected && self.state.mousePressed) {
+        app.ctx.save()
+        const mouse = app.eventToMouse(e)
+        const fonts = ['Times New Roman', 'Arial', 'monospace', 'Comic Sans', 'fantasy']
+        let randFont = Math.floor(Math.random() * fonts.length)
+        let randSize = Math.floor(Math.random() * 70) + 15
+        // draw random integers between 0-9
+        app.ctx.font = randSize + 'px ' + fonts[randFont]
+        const rFill = Math.random() * 255
+        const gFill = Math.random() * 255
+        const bFill = Math.random() * 255
+        app.ctx.fillStyle = `rgb(${rFill}, ${gFill}, ${bFill})`
+        const rStroke = Math.random() * 255
+        const gStroke = Math.random() * 255
+        const bStroke = Math.random() * 255
+        app.ctx.strokeStyle = `rgb(${rStroke}, ${gStroke}, ${bStroke})`
+        //const randNum = Math.floor(Math.random() * (10))
```

Comment:
the StandardJS linter should also be throwing an error here, it prefers comments include a space after the `//` && before the `const`... although, unless u pan on doing something w/this line of code later i would just delete it